### PR TITLE
Get current session

### PR DESCRIPTION
**Pull request by itaihanski**
## Related Issues

Fixes https://github.com/descope/etc/issues/7931

## Description

- Add `isSessionLoading` flag for initial loading
- Add current session, refresh and user accessors
- Bump version to `0.7.0`

## Must

- [x] Tests
- [x] Documentation (if applicable)

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -3,4 +3,6 @@ export { default as useDescope } from './hooks/useDescope'
 export { default as useSession } from './hooks/useSession'
 export { default as useFlow } from './hooks/useFlow'
 
+export { getCurrentSessionToken, getCurrentRefreshToken, getCurrentUser } from './helpers'
```
Comment (asafshen) [inline src/index.tsx:6]: it may be nice to add those to readme as well

File: src/hooks/useSession.ts [hunk 2]
```diff
@@ -23,21 +34,27 @@ const useSession = (): DescopeSessionManager => {
...
     setSession(updatedSession)
   }
 
   const updateTokens = async (sessionJwt: string, refreshJwt: string) => {
     if (session) {
       logger?.log('updating current session with new tokens')
-      const updateSession = { ...session, sessionJwt, refreshJwt }
-      await DescopeReactNative.saveItem(projectId, JSON.stringify(updateSession))
-      setSession(updateSession)
+      const updatedSession = { ...session, sessionJwt, refreshJwt: refreshJwt || session.refreshJwt }
```
Comment (asafshen) [inline src/hooks/useSession.ts:45]: is there a scenario that someone calls updateTokens with only session token? (e.i. empty refreshJwt)

**Review by asafshen — DISMISSED** [review]
looks good

not sure if we have testing / docs for this repo, we can add it later on

File: src/hooks/useSession.ts (hunk 2 above)
Comment (itaihanski) [reply src/hooks/useSession.ts:45]: It's an edge case. In any way - the current session's refresh token is still viable

**Pull request merged**